.pre-commit-config.yaml

```diff
@@ -21,8 +21,8 @@ repos:
     rev: v1.12.1
     hooks:
       - id: blacken-docs
-        additional_dependencies: [black==22.8.0]
-        args: [-l, '79', -t, py39]
+        additional_dependencies: [black==22.10.0]
+        args: [-l, '77', -t, py39]
```

Comment:
This is a hack, but without it blacken-docs was fighting with flake8 by creating a line longer than 80 characters in a docstring. I think this is a bug in blacken-docs, although maybe I misunderstand it.

Reply:
I don't have any clue on this. The work around works for me though :)